spec/models/participant_spec.rb

```diff
@@ -20,4 +20,27 @@
       end
     end
   end
+  describe 'total score' do
+    it 'should be zero' do
```

Comment:
It's difficult to know from this description under what conditions it would be 0.